superbench/benchmarks/micro_benchmarks/memory_bw_latency_performance.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module for running the Intel MLC tool to measure memory bandwidth and latency."""
+
+import os
+from pathlib import Path
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, ReturnCode
+from superbench.benchmarks.micro_benchmarks import MicroBenchmarkWithInvoke
+
+
+class MemBwLatencyBenchmark(MicroBenchmarkWithInvoke):
+    """The Memory bandwidth and latency benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'mlc'
+        self.__support_mlc_commands = ['bandwidth_matrix','latency_matrix','max_bandwidth']
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
+
+        self._parser.add_argument(
+                '--tests',
+                type=str,
+                nargs='+',
+                default=["bandwidth_matrix"],
+                required=False,
+                help='The modes to run mlc with. Possible values are {}.'.format(' '.join(self.__support_mlc_commands))
+        )
+
+    def _preprocess(self):
+        """Preprocess/preparation operations before the benchmarking.
+
+        Return:
+            True if _preprocess() succeed.
+        """
+        if not super()._preprocess():
+            return False
+
+        mlc_path = os.path.join(self._args.bin_dir,self._bin_name)
+        ret_val = os.access(mlc_path, os.X_OK|os.F_OK)
+        if ret_val == False:
+            logger.error('Executable {} not found in {} or it is not executable'.format(self._bin_name,self._args.bin_dir))
+            return False
+
+        # the mlc command requires hugapage to be enabled
+        self._commands.append('bash -c "echo 4000 > /proc/sys/vm/nr_hugepages"')
```

Comment:
I am not sure whether we should do it here. @abuccts and @guoshzhao, what's suggestion on this?